docs/pages/bare/hello-world.md

```diff
@@ -5,13 +5,15 @@ sidebar_label: Up and Running
 
 > This guide assumes that you have Xcode and/or Android Studio installed and working. It also assumes that you want to create a new project. If you have an existing app that you would like to integrate the Expo SDK in an existing app, read the [existing apps guide](/bare/existing-apps/).
 
-To get started with a bare React Native project, run `expo init` and choose one of the bare templates. We'll use the minimum template here.
+Before you get started with a bare React Native project, make sure you set up your environment for [React Native CLI](https://reactnative.dev/docs/environment-setup).
+
+After this, let's get started with a bare project. Run `expo init` and choose one of the bare templates. We'll use the minimum templat ehere.
 
 ```sh
+# If you don't have @react-native-community/cli, install it and check out https://reactnative.dev/docs/environment-setup
+npm i -g @react-native-community/cli
```

Comment:
they actually recommend using `npx` these days so we can probably remove this

Reply:
Hmm, we might want to help them setup NCC (like `create-react-native-app`). Waiting almost 30s on a fast MacBook is terrible DX :|
![Screenshot 2020-10-07 at 20 13 50](https://user-images.githubusercontent.com/1203991/95370824-aa1ec200-08d9-11eb-939b-1a0f4305b2c8.png)